xmtp_mls/src/client.rs

```diff
@@ -201,6 +201,17 @@ where
         self.identity.installation_keys.to_public_vec()
     }
 
+    /// Get the inbox_id associated with the client
```

Comment:
Just hacking these in for now until we have the new Identity struct on the client